system/jhtree/jhtree.cpp

```diff
@@ -1532,12 +1529,12 @@ bool CKeyCursor::_next(KeyStatsCollector &stats)
     {
         node.setown(key.locateFirstNode(stats));
         nodeKey = 0;
-        return node && node->getKeyAt(nodeKey, keyedBuffer);
+        return node && node->isKeyAt(nodeKey);
```

Comment:
CKeyCursor::next() also needs to get the key after calling _next

Reply:
Oh no, I would have broken keydiff....

Reply:
Fixed